changelogs/fragments/5143-fix-xfs-project-quota-initialization.yaml

```diff
@@ -0,0 +1,3 @@
+---
+bugfixes:
+  - xfs_quota - the call to "xfs_quota" to set up project quotas was missing the project name and hence failed (https://github.com/ansible-collections/community.general/issues/5143).
```

Comment:
```suggestion
  - xfs_quota - the call to ``xfs_quota`` to set up project quotas was missing the project name and hence failed (https://github.com/ansible-collections/community.general/issues/5143).
```

Reply:
Will do. Out of curiosity, why is one quotation mark style preferred over the other one? Do these fragments get parsed later with some script/tool which needs this kind of style?

Reply:
The strings in these fragments are interpreted as [reStructuredText](https://en.wikipedia.org/wiki/ReStructuredText); they end up in CHANGELOG.rst (for exmaple https://github.com/ansible-collections/community.general/blob/stable-5/CHANGELOG.rst).

Reply:
Thanks for the explanation! I read about .rst files but never worked with them so far.